universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -236,10 +242,14 @@ private ContainerTag getCheckboxContainer() {
   }
 
   private Tag buildFieldErrorsTag() {
+    String[] referenceClasses =
+        referenceClassesBuilder.build().stream()
+            .map(ref -> (ref + "-error").toString())
```

Comment:
(blocking) I don't think you need the `toString()`.

Reply:
Done.